src/main/java/igym/config/ValidationConfig.java

```diff
@@ -13,7 +13,7 @@ public class ValidationConfig {
     static public Validator validator() {
         ValidatorFactory factory = Validation.byDefaultProvider()
                 .configure()
-                .addProperty("hibernate.validator.fail_fast", "true")  
+                .addProperty("hibernate.validator.fail_fast", "false")  
                 .buildValidatorFactory();
         return factory.getValidator();
     }
```

Comment:
I have deleted all these files, with your solution I think it doesn't make a difference which exception Hibernate will throw first, we are checking if both are threw